avocado/core/parser.py

```diff
@@ -135,9 +134,6 @@ def start(self):
         # Allow overriding default params by plugins
         variants = varianter.Varianter(getattr(self.args, "varianter_debug", False))
         self.args.avocado_variants = variants
-        # FIXME: Backward compatibility params, to be removed when 36 LTS is
-        # discontinued
-        self.args.default_avocado_params = tree.TreeNode()
```

Comment:
Before doing this we have to fix `avocado-virt`, which is still using the `default_avocado_params`.

Reply:
Before merging this, sure.  I'm heading to review your fixes for Avocado-Virt.